script.js

```diff
@@ -0,0 +1,235 @@
+/*------ M O D A L ------*/
+var modal = {
+  currentSlide: null,
+  autoPlay: false,
+  SLIDE_TIMEOUT: 2000,
+  slides: [
+    {
+      idx: 0,
```

Comment:
is the idx necessary? I think we could infer it from the index

Reply:
I use it for ```getNextSlide()``` and ```getPrevSlide()```